### PR TITLE
What-is and TOC clean-up

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: docs/sphinx/_toc.yml.in [hunk 1]
```diff
@@ -174,8 +140,22 @@ subtrees:
           title: ROCm SMI LIB
         - url: https://rocm.docs.amd.com/projects/ROCmValidationSuite/en/latest/
           title: ROCm validation suite
```
Comment (Rmalavally) [inline docs/sphinx/_toc.yml.in:142]: ROCm Validation Suite is not a library. Perhaps move it to Install?  

@LisaDelaney

File: docs/sphinx/_toc.yml.in [hunk 2]
```diff
@@ -174,8 +140,22 @@ subtrees:
...
-        - url: https://rocm.docs.amd.com/projects/roctracer/en/latest/index.html
-          title: ROCTracer
+        - url: https://rocm.docs.amd.com/projects/rocPRIM/en/latest/index.html
+          title: rocPRIM
+        - url: https://rocm.docs.amd.com/projects/rocprofiler/en/latest/rocprof.html
+          title: ROCProfiler
```
Comment (Rmalavally) [inline docs/sphinx/_toc.yml.in:146]: Move ROCProfiler and ROCTracer under Tools for ease of access? @LisaDelaney

File: docs/what-is-rocm.md [hunk 3]
```diff
@@ -58,7 +57,6 @@ ROCm consists of the following drivers, development tools, and APIs.
 | [rocBLAS](https://rocm.docs.amd.com/projects/rocBLAS/en/latest/)| A BLAS implementation (in the HIP programming language) on the ROCm runtime and toolchains |
 | [rocFFT](https://rocm.docs.amd.com/projects/rocFFT/en/latest/) | A software library for computing fast Fourier transforms (FFTs) written in HIP |
 | [ROCK-Kernel-Driver](https://github.com/RadeonOpenCompute/ROCK-Kernel-Driver/) | An AMDGPU Driver with KFD that is used by ROCm |
-| [ROCm Augmentation Library (rocAL)](https://rocm.docs.amd.com/projects/rocAL/en/latest/doxygen/html/index.html) | An augmentation library designed to decode and process images and videos |
 | [ROCmCC](https://rocm.docs.amd.com/en/latest/reference/rocmcc/rocmcc.html) | A Clang/LLVM-based compiler |
 | [ROCm cmake](https://github.com/RadeonOpenCompute/rocm-cmake) | A collection of CMake modules for common build and development tasks |
 | [ROCm Data Center Tool](https://rocm.docs.amd.com/projects/rdc/en/latest/) | Simplifies administration and addresses key infrastructure challenges in AMD GPUs in cluster and data-center environments |
```
Comment (Rmalavally) [inline docs/what-is-rocm.md:62]: Also under ROCm Tools @LisaDelaney

File: docs/what-is-rocm.md [hunk 4]
```diff
@@ -23,10 +23,9 @@ source software compilers, debuggers, and libraries. ROCm is fully integrated in
 
 ROCm consists of the following drivers, development tools, and APIs.
 
-| ROCm product | Description |
+| Project | Description |
```
Comment (Rmalavally) [inline docs/what-is-rocm.md:26]: These are not projects but libraries. Do we want to rename this to Libraries?

File: docs/what-is-rocm.md (hunk 3 above)
Comment (Rmalavally) [reply docs/what-is-rocm.md:62]: Sounds good, Lisa. Previously, we have had rocPRIM with other Math and Communication Libraries. You could consider moving it under Reference/Libraries.

@LisaDelaney

File: docs/what-is-rocm.md [hunk 5]
```diff
@@ -58,13 +57,12 @@ ROCm consists of the following drivers, development tools, and APIs.
...
 | [ROCmCC](https://rocm.docs.amd.com/en/latest/reference/rocmcc/rocmcc.html) | A Clang/LLVM-based compiler |
 | [ROCm cmake](https://github.com/RadeonOpenCompute/rocm-cmake) | A collection of CMake modules for common build and development tasks |
 | [ROCm Data Center Tool](https://rocm.docs.amd.com/projects/rdc/en/latest/) | Simplifies administration and addresses key infrastructure challenges in AMD GPUs in cluster and data-center environments |
 | [ROCm Debug Agent Library (ROCdebug-agent)](https://github.com/ROCm-Developer-Tools/rocr_debug_agent/) | A library that can print the state of all AMD GPU wavefronts that caused a queue error by sending a SIGQUIT signal to the process while the program is running |
 | [ROCm Debugger (ROCgdb)](https://rocm.docs.amd.com/projects/ROCgdb/en/latest/) | A source-level debugger for Linux, based on the GNU Debugger (GDB) |
-| [ROCm Debugger API (ROCdbgapi)](https://rocm.docs.amd.com/projects/ROCdbgapi/en/latest/) | The ROCm debugger library |
+| [ROCdbgapi](https://rocm.docs.amd.com/projects/ROCdbgapi/en/latest/) | The ROCm debugger library |
```
Comment (saadrahim) [inline docs/what-is-rocm.md:65]:
```suggestion
| [ROCdbgapi](https://rocm.docs.amd.com/projects/ROCdbgapi/en/latest/) | The ROCm debugger API library |
```

**Review by saadrahim — CHANGES_REQUESTED** [review]

**Review by saadrahim — DISMISSED** [review]

File: docs/sphinx/_toc.yml.in [hunk 6]
```diff
@@ -88,94 +88,73 @@ subtrees:
...
         - url: https://rocm.docs.amd.com/projects/ROCgdb/en/latest/index.html
-          title: ROCm debugger AP
+          title: ROCm debugger API
         - url: https://rocm.docs.amd.com/projects/rocm_smi_lib/en/latest/
           title: ROCm SMI LIB
         - url: https://rocm.docs.amd.com/projects/ROCmValidationSuite/en/latest/
           title: ROCm validation suite
-        - url: https://rocm.docs.amd.com/projects/roctracer/en/latest/index.html
-          title: ROCTracer
+        - url: https://rocm.docs.amd.com/projects/rocPRIM/en/latest/index.html
```
Comment (saadrahim) [inline docs/sphinx/_toc.yml.in:142]: if the version of ROCm changes, these links will always point to latest.

**Review by saadrahim — APPROVED** [review]

**Pull request merged**